src/main/java/no/nav/fo/veilarbregistrering/registrering/tilstand/Status.java

```diff
@@ -25,7 +25,6 @@
     UKJENT_TEKNISK_FEIL("ukjentTeknisk"),
     TEKNISK_FEIL("teknisk"),
     OPPGAVE_OPPRETTET("oppgaveOpprettet"),
```

Comment:
Jeg lurer på om `OPPGAVE_OPPRETTET` også kan slettes. Den brukes for oppgaver som ble vellykket opprettet ifm. registreringsflyten - ikke den synkrone jobben bruker selv initierer.